tests/gold_tests/ip_allow/run_sed.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# The first sed command in the pipeline eliminates the first 3 log fields from each log line.
+
+sed 's/^[^ ]* [^ ]* [^ ]* //' < $1 | sed 's/:[0-9][0-9]*\([^0-9]\)/:SOMEPORT\1/g'
```

Comment:
I tried various ways of putting this command directly into ip_allow.test.py, but no luck.